aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -854,12 +857,35 @@ Tensor normal(double mean, double std, IntArrayRef size,
   return result.normal_(mean, std, generator);
 }
 
+Tensor normal_meta(double mean, double std, IntArrayRef size,
+                   c10::optional<Generator> generator,
+    c10::optional<ScalarType> dtype,
+    c10::optional<Layout> layout,
+    c10::optional<Device> device,
+    c10::optional<bool> pin_memory) {
+  CHECK_NORMAL_STD(std);
```

Comment:
I think it's a good idea to call both `mul` and `add` here (and in the others). Otherwise we won't have `mean` and `std` shape/dtype/broadcasting checks.

@bdhirsh What do you think? I'd say this is needed since this kernel is hybrid (i.e. half-composite).